Example/YoshiExample/AppDelegate.swift

```diff
@@ -37,30 +37,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let menuItemQA = MenuItem(name: "QA")
         let environmentItems: [YoshiTableViewMenuItem] = [menuItemProd, menuItemStaging, menuItemQA]
 
-        let tableViewMenu = TableViewMenu(debugMenuName: "Environment",
+        let tableViewMenu = TableViewMenu(name: "Environment",
+            value: nil,
             displayItems: environmentItems, didSelectDisplayItem: { (displayItem) in
             NSNotificationCenter.defaultCenter()
-                .postNotificationName(Notifications.EnvironmentUpdatedNotification, object: displayItem.displayText())
+                .postNotificationName(Notifications.EnvironmentUpdatedNotification, object: displayItem.displayText)
         })
 
         // YoshiMenuType.DateSelector
-        let dateSelector = DateSelector(debugMenuName: "Environment Date", didUpdateDate: { (dateSelected) in
+        let dateSelector = DateSelector(name: "Environment Date",
+            value: nil,
```

Comment:
It's used as the subtitle text in the table view cell. I imagine users can display the "current value" for osmething there before opening that menu.